aten/src/ATen/native/quantized/cpu/qfc.cpp

```diff
@@ -0,0 +1,152 @@
+#include <ATen/ATen.h>
+#include <ATen/core/op_registration/op_registration.h>
+#include <ATen/cpp_custom_type_hack.h>
+#include <ATen/fbgemm_utils.h>
+
+#include <algorithm>
+#include <tuple>
+
+namespace at {
+namespace native {
+namespace {
+
+class QFCInt8 final : public c10::OperatorKernel {
+ public:
+#ifdef USE_FBGEMM
+  std::tuple<at::Tensor, double, int64_t> operator()(
+      const at::Tensor& input,
+      double input_scale,
```

Comment:
Is using double for scale convention in PT quantized operator? Otherwise, we can just use float here and we won't need the static_cast below.

Reply:
Sorry, just saw that in the follow-up diff that makes my comment irrelevant. Please ignore.